tests/API/Header/HeaderTest.php

```diff
@@ -40,4 +41,14 @@ public function testContentType()
         $this->assertEquals($contentType, $header->getValue());
         $this->assertEquals("Content-Type: $contentType\n", $header->get());
     }
+
+    public function testTelemetry()
+    {
+        $telemetry = uniqid();
```

Comment:
if you're keeping this class, consider renaming the var to something different than the class name. e.g. "data" or "values"